Makefile.toml

```diff
@@ -100,32 +100,26 @@ rm -rf ${DOCS_FOLDER}/*
 dependencies = ["docs-clean", "schema"]
 description = "Generate documentation"
 script = '''
-echo "🔎 Checking jsonschema2md installed..."
-if ! which jsonschema2md >/dev/null;
-then
-    echo "\n❌ jsonschema2md could not be found"
-    echo "  Consider installing jsonschema2md to generate documentation.\n"
-    echo "＞ \`npm install -g @adobe/jsonschema2md\`\n\n"
-    exit 1
-fi
+echo "📦 Installing fadroma-schema..."
+
+npm install --save-dev @fadroma/schema@1.0.5
```

Comment:
If you add a new tool (such as `npm` or `npx`) to the environment, don't forget to reference it in the [README.md#pre-requisites](https://github.com/okp4/contracts#pre-requisites) section. :relieved: